test/samples/complex/input.json

```diff
@@ -1,18 +1,22 @@
 {
-	"id": "http://yourdomain.com/schemas/myschema.json",
-	"description": "Complex example",
-	"type": "object",
-	"properties": {
-		"address": {
-			"type": "object",
-			"properties": {
-				"street_address": {
-					"type": "string"
-				},
-				"country": {
-					"type" : "string"
-				}
-			}
-		}
-	}
+    "id": "http://yourdomain.com/schemas/myschema.json",
+    "description": "Complex example",
+    "allOf": [
```

Comment:
We dont need to add allOf here as we have other examples that do this, can we revert this please.